src/components/Sidebar/components/ExpandedList/ExpandedList.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { ReactNode } from "react";
+import styled from "styled-components/macro";
+import { Color } from "variables";
+
+type Props = {
+  children: ReactNode;
+};
+
+// Defines the component plus it's css
+const StyledExpandedList = styled.div`
+  position: absolute;
+  left: 0;
+  right: 0;
+  margin: 3px 0;
+  padding: 4px 0;
+  border-radius: 3px;
+  background-color: ${Color.White};
+  text-align: left;
+
+  a {
+    display: block;
+    padding: 11px 15px;
+    text-decoration: none;
+    font-size: 15px;
+    font-weight: 500;
+    color: ${Color.CodGray};
+    transition: color 0.2s ease-in-out, background-color 0.2s ease-in-out;
+  }
+
+  a:hover {
+    color: white;
+    background-color: ${Color.DodgerBlue};
+  }
+`;
+
```

Comment:
Move your styles to this component
```suggestion
export const ExpandedListItem = styled.span`
    display: block;
    padding: 11px 15px;
    text-decoration: none;
    font-size: 15px;
    font-weight: 500;
    color: ${Color.CodGray};
    transition: color 0.2s ease-in-out, background-color 0.2s ease-in-out;
    
    &:hover {
      color: white;
      background-color: ${Color.DodgerBlue};
    }
`
```

Reply:
Then you'd use it like this
```TypeScript
import ExpandedList, { ExpandedListItem } from '..'
```

```jsx
<ExpandedList>
  <ExpandedListItem>
    <Link to="">LinkOne</Link>
  </ExpandedListItem>
  
  <ExpandedListItem>
    <Link to="">LinkTwo</Link>
  </ExpandedListItem>


  <MySpecialThingInTheList>
    Who knows
  </MySpecialThingInTheList>
</ExpandedList>
```

Reply:
And this way, we could later add even props to the items, that might behave differently
```jsx
  <ExpandedListItem deactivated>
    <Link to="">LinkTwo</Link>
  </ExpandedListItem>
```

Reply:
Right so some questions :/

I add the ExpandedListItem to ExpandedList.tsx but I don't use it there? It is to be used by importing it in another file for example Sidebar.tsx
Then it is to be populated with whatever we come up with. So in your example Link is another component altogether?

Reply:
Correct. They just live together in that file cause they are so closely related. 
The main difference is the export:
- `export const ExpandedListItem`
- `export default ExpandedList`

Reply:
Thanks @MoSattler hopefully I have moved this in the right direction now.